weave/pantry.py

```diff
@@ -35,7 +36,8 @@ class Pantry():
     """Facilitate user interaction with the index of a Weave data warehouse.
     """
 
-    def __init__(self, index: IndexABC, pantry_path="weave-test", **kwargs):
+    def __init__(self, index: IndexABC, pantry_path="weave-test",
+                 use_mongo=False, **kwargs):
```

Comment:
Maybe instead of taking a new parameter, we check if we have access to mongo when we call upload_basket and then call the load_mongo by default if its available

Reply:
At first, that was a one option I had in mind. But, then again let's let the user decide.

Reply:
Ok, I actually agree with both of you. I think the default should be `True`, but the user can ignore if need be.

Reply:
clicking on that 'merge pull request' is so satisfying

Reply:
> Ok, I actually agree with both of you. I think the default should be `True`, but the user can ignore if need be.
> 
> Note that I think this means we should also check for mongo availability.

I like this. Default the kwarg to True, but still have a check to ensure we can access pymongo before using the mongo loader

Reply:
per Ken's request, I changed the option to use mongodb client instead of a boolean